translations/ui-orders/en.json

```diff
@@ -667,6 +667,11 @@
   "settings.setPOLInesLimit.validation": "Please enter a whole number greater than 0 and less than 1000 to continue",
   "settings.setPOLInesLimit": "Set purchase order lines limit",
 
+  "settings.acquisitionMethods": "Acquisition methods",
+  "settings.acquisitionMethods.singular": "Acquisition method",
+  "settings.acquisitionMethods.name": "Name",
+  "settings.acquisitionMethods.validation.name": "Acquisition method's name is invalid",
```

Comment:
Where do you use it?

Reply:
oh, this line remained from the previous implementation, I'll remove it, thanks